chatlearn/models/vllm_module_v2.py

```diff
@@ -160,21 +149,7 @@ def _get_sampling_params(self, is_eval):
             sampling_params.use_beam_search = self.model_args.get("use_beam_search")
         return sampling_params
 
-    @property
-    def data_parallel_size(self):
-        """
-        :meta private:
-        """
-        return 1
-
-    @property
-    def data_parallel_rank(self):
-        """
-        :meta private:
-        """
-        return 0
-
```

Comment:
What's the reason for deleting these codes?

Reply:
I have added them back to VLLMWokerWrapper